File: luke17.py
def endreTall(tall):
    nyttTall = int("6" + str(tall)[:-1])
    return nyttTall

n = 6
while True:
    tall = n
    nyttTall = endreTall(tall)
    if nyttTall / 4 == tall:
        print("Done! ", tall, nyttTall)
        break
    n += 10

File: luke17.java

public class luke17 {
    public static void main(String[] args){

        int tall = 6;

        while(true) {
            if (flyttTall(tall) / 4 == tall) {
                System.out.println("Done!" + " " + tall + " " + flyttTall(tall));
                break;
            }
            tall += 10;
        }    
    }
    
    static int flyttTall(int tall) {
        String nyttTall = Integer.toString(tall);
        nyttTall = "6" + nyttTall.substring(0, nyttTall.length()-1);
        int nyttTall2 = Integer.valueOf(nyttTall);
        return nyttTall2;
    }

}